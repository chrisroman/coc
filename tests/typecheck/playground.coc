[fn: [a: *][x: a]a]*
