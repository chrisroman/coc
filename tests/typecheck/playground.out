[fn: [a: *][x: a]a]*
Type: *
